CHANGELOG.md

```diff
@@ -6,10 +6,28 @@
 
 ### Breaking changes 💔
```

Comment:
Should we add a changelog entry for this PR under `Improvements`? Something about how the remote device now uses the Qiskit `Primitive` and `Session` functionality?

Reply:
Hmm... is that an improvement or a feature? I'm surprised `qiskit_session` isn't listed as a feature actually, must have missed adding it in the actual PR. I think I'll add `qiskit_session` as a feature and Primitives as an improvement to the changelog.